cmd/meroxa/root/connectors/logs.go

```diff
@@ -83,7 +83,7 @@ func (l *Logs) Logger(logger log.Logger) {
 	l.logger = logger
```

Comment:
In execute, `GetConnectorLogs` should now accept nameOrID https://github.com/meroxa/meroxa-go/pull/78/files#diff-c10a5335e8fa840c34013ae4363be88f6b8c7ee5573945ced446d6a5f4afe5b5R11